### PR TITLE
My first pull request

### PR DESCRIPTION
**Pull request by ch4ne5teban**
This is a "Hello world" repository to master all of the tools and best practices related to Git and GitHub part of Module 1 Block 1. Consists of an index.html and style.css files. Thank you in advance for your review.

### CODE REVIEW

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @ch4ne5teban  ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Excellent PR title and description
- [x] Good commit history
- [x] Proper use of `.gitignore` file
- [x] Correct GitHub Flow  

## Required Changes ♻️

- [ ] You need to properly install linters as requested in the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md). You do have the `.github` folder (kudos!) but you're missing `webhint` and `stylelint`. I suggest you follow [this](https://github.com/microverseinc/linters-config/blob/master/html-css/README.md) step-by-step guide on how to install them.
- [ ] After installing linters, make sure all checks are passing.

_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by vechicin** [comment]
https://github.com/ch4ne5teban/hello-microverse/blob/d783b6ee4b718d7e9a0b806344fc91d5d405b71f/README.md?plain=1#L12-L26
I suggest you add instructions on how to get a local copy of the repository and how to install the necessary dependencies for the project to run properly.

**Comment by vechicin** [comment]
https://github.com/ch4ne5teban/hello-microverse/blob/d783b6ee4b718d7e9a0b806344fc91d5d405b71f/README.md?plain=1#L28-L33
Make sure you link your social media accounts properly in this section. This will be tremendously helpful when job hunting because it'll allow anyone interested in your projects to directly go to your social media profiles and get in touch with you!

File: style.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+*{
+  background-color: yellowgreen;
+}
+
+h1{
+  color: whitesmoke;
+}
```
[the remark this answers is not in the document]
Comment (ch4ne5teban) [reply style.css]: Thank you. I've switched to hex equivalents.

**Comment by ch4ne5teban** [comment]
Really appreciate your detailed feedback :)

**Review by omarsalem7 — APPROVED** [review]
Hi @ch4ne5teban , 
# STATUS: APPROVED! 🥇 💯 
giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors ✔️

Cheers and Happy coding!👏👏👏

## Optional suggestions

- [ ] [OPTIONAL] kindly add a descriptive title for your pull request for what this PR contains.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Code Review by @omarsalem7

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -29,8 +28,8 @@ To get a local copy up and running follow these simple example steps.
 
 👤 Esteban
 
-- GitHub: @ch4ne5teban
-- Twitter: @ch4ne5teban
+- GitHub: @ch4ne5teban (https://github.com/ch4ne5teban)
+- Twitter: @ch4ne5teban (https://twitter.com/ch4ne5teban)
```
[the remark this answers is not in the document]
Comment (ch4ne5teban) [reply README.md:32]: Shokran, I'll keep that in mind :)

**Pull request merged**